site-manage/site-manage-participant-helper/src/java/org/sakaiproject/site/tool/helper/participant/impl/SiteAddParticipantHandler.java

```diff
@@ -1265,37 +1266,6 @@ protected void updateOfficialAccountParticipant(String officialAccount, User u,
 	}
 	
 	/**
-	 * generate a 9 digit password. Chars are randomly picked from lower-case/upper-case alpha lists, a numerical list, and a symbol list
-	 * @return
-	 */
-	protected String generatePassword()
-	{
-		// set random password
-		int length = 9;
-		StringBuffer rndbuf = new StringBuffer(length);
-		for (int i = 0; i < length; ++i){
-			int chooseArray = (int) (4 * Math.random());
-		    switch (chooseArray) {
-		    	case 0: 
-		    		rndbuf.append(LOWER_ALPHA_ARRAY[(int) ( LOWER_ALPHA_ARRAY.length * Math.random())]);
-		    		break;
-		    	case 1:
-		    		rndbuf.append(UPPER_ALPHA_ARRAY[(int) ( UPPER_ALPHA_ARRAY.length * Math.random())]);
-		    		break;
-		    	case 2:
-		    		rndbuf.append(NUMBER_ARRAY[(int) ( NUMBER_ARRAY.length * Math.random())]);
-		    		break;
```

Comment:
Also forgot to remove these character/number/symbol arrays, as they're no longer used.